### PR TITLE
feat(Releases): Improve test coverage to suspect commits.

### PR DESCRIPTION
**Pull request by lauryndbrown**

### CODE REVIEW

File: tests/sentry/utils/test_committers.py [hunk 1]
```diff
@@ -2,25 +2,160 @@
...
+    def test_dot_does_not_separate(self):
+        assert list(tokenize_path('foo.bar')) == ['foo.bar']
+
+    def test_additional_slash_in_front(self):
+        assert list(tokenize_path('/foo/bar')) == ['bar', 'foo']
+        assert list(tokenize_path('\\foo\\bar')) == ['bar', 'foo']
+
+    def test_additional_slash_with_dot_in_front(self):
+        # TODO(lb): better name for this?
+        # TOOD(lb): Also.... I'm not sure we should be capturing '.'
```
Comment (markstory) [inline tests/sentry/utils/test_committers.py]: You could call this `with_relative_paths`

File: tests/sentry/utils/test_committers.py [hunk 2]
```diff
@@ -2,25 +2,160 @@
...
+        assert list(tokenize_path('./')) == ['.']
+        assert list(tokenize_path('./../')) == ['..', '.']
```
Comment (markstory) [inline tests/sentry/utils/test_committers.py]: Commit paths shouldn't have `.` or `..` in them, but they could show up in stacktraces in theory. Given we won't have access to the rest of the file path capturing the dots is the best we can do.

**Comment by lauryndbrown** [comment]
I'm going to call this pr done here, I believe this helps significantly with the coverage. There were two TestCases that I mocked out but did not get to complete: 
```
class GetCommittersTestCase(CommitTestCase):
    def setUp(self):
        super(GetCommittersTestCase, self).setUp()

    def test_get_commits_committer(self):
        pass

    def test_simple(self):
        pass


class GetEventFileCommittersTestCase(TestCase):
    pass
```

I left a TODO in the test class to add more test for those methods.

**Review by markstory — APPROVED** [review]
:+1: for having more tests on this code.

**Pull request merged**